tslint.json

```diff
@@ -0,0 +1,29 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended"
+    ],
+    "exclude": [ "./src/languages/*.ts"],
```

Comment:
What are the reasons for excluding the files in the `languages` directory.

Reply:
Also, I think you should exclude the `lib` directory since that code is not part of this project.